cvmfs/catalog_mgr.h

```diff
@@ -217,6 +217,7 @@ class AbstractCatalogManager : public SingleCopy {
                                 shash::Any   *catalog_hash) = 0;
   virtual void UnloadCatalog(const CatalogT *catalog) { }
   virtual void ActivateCatalog(CatalogT *catalog) { }
+  const std::vector<CatalogT*>& GetCatalogs() const { return catalogs_; }
```

Comment:
Yes, here:
```
void WritableCatalogManager::Balance() {
  CatalogList catalog_list = GetCatalogs();
  for (unsigned i = 0; i < catalog_list.size(); ++i) {
    FixWeight(catalog_list[i]);
  }
}
```
I need to get the catalog list in order to balance them. I though it was better to return a constant reference instead of directly using the private attribute.